docs/source/documentation.rst

```diff
@@ -155,13 +155,28 @@ GPU-compatible installation
 ---------------------------
 .. NOTE :: This feature is not available if you are using a macOS.
 
+Linux
+~~~~~~~
+
 By default, AxonDeepSeg installs the CPU version of TensorFlow. To train a model using your GPU, you need to uninstall the TensorFlow from your virtual environment, and install the GPU version of it::
 
     pip uninstall tensorflow
```

Comment:
Should we change the instructions to conda here too?

Reply:
I think it's a good idea.